### PR TITLE
Split CI zkg run into separate test and install steps.

### PR DESCRIPTION
**Pull request by rsmmr**
Installing with --force lets it ignore errors.

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -15,13 +15,19 @@ package_ci_task:
     cpu: 2
     memory: 12G
 
+  test_script:
+    - zkg test .
+
   install_script:
-    - zkg install --force .
+    - zkg install --force --skiptests .
```
Comment (bbannier) [inline .cirrus.yml:22]: I am confused now why we install with `--force` if it in turn forces us to deviate from the installation method a user would default to (ideally we'd mirror what a user would do during installation as much as possible and only add our own steps _in addition to that_).

Could you add a comment somewhere here capturing the reasoning?

**Review by bbannier — APPROVED** [review]

File: .cirrus.yml (hunk 1 above)
Comment (rsmmr) [reply .cirrus.yml:22]: "Force" avoids interactive prompts ("do you want to install package X?").

Comment (bbannier) [reply .cirrus.yml:22]: Hmm, that's unfortunate. Could we maybe instead work around that with e.g.,

    $ yes | zkg install .
    $ echo Y | zkg install .

I think I tried using `yes` previously when this CI also had other issues, so maybe we need to work around issues with `echo Y`.

Not something super critical which needs to block this PR, but IMO still nice to have.

Comment (rsmmr) [reply .cirrus.yml:22]: not too tempted to try that :) I think the force is fine

**Pull request merged**